style.css

```diff
@@ -0,0 +1,730 @@
+*,
+::before,
+::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'lato', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.black-bg {
+  background-color: black;
+}
+
+.black-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 90%;
+  color: white;
+  list-style: none;
+  gap: 20px;
+  padding: 10px 50px;
+}
+
+.black-list a {
+  color: white;
+}
+
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 75%;
+  margin: auto;
+}
+
+.brand-name {
+  color: black;
+  font-weight: bold;
+  font-size: 20px;
+}
+
+.brand-name a {
+  color: black;
+}
+
+.nav {
+  margin-top: 12px;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  gap: 30px;
+}
+
+.nav-links a {
+  color: black;
+}
+
+.nav-item {
+  padding: 5px;
+  border: 3px solid white;
+}
+
+.nav-itemc {
+  border: 3px solid red;
+  color: #ff5438;
+  padding: 5px;
+}
+
+.nav-itemc a {
+  color: #ff5438;
+}
+
+/** CSS for Toggle menu **/
+
+.ham-icon {
+  display: block;
+  cursor: pointer;
+}
+
+.bars {
+  display: none;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background: black;
+}
+
+.hide {
+  visibility: hidden;
+}
+
+.banner-area {
+  background-image: url('images/banner.png');
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-size: cover;
+  width: 100%;
+}
```

Comment:
- [ ] As mentioned previously the width in the About page is overflowing and creating a scroll behavior so I'd advise you fix it as it leads to a bad UX.
- Here's how it looks:
  ![FtIBjCsD1R](https://github.com/nouman17610/developers-meetup/assets/104968236/9206f053-aff4-46e2-ad14-abeddecd4fd7)